udata_front/theme/gouvfr/datagouv-components/src/components/AppLink/AppLink.stories.ts

```diff
@@ -5,6 +5,7 @@ import { vueRouter } from 'storybook-vue3-router';
 const meta = {
   title: 'Components/AppLink',
   component: AppLink,
+  decorators: [vueRouter()],
```

Comment:
If we want to put it here for all stories, we should somehow remove it for the stories without vue-router